examples/sampling-slice-rank-shrinking-mcmc.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
This notebook needs to introduce the topic (and link to the docs page): Doesn't mention slice sampling at all now until the 2nd section!

Reply:
Have updated this and streamlined the notebook